plugins/modules/win_firewall.ps1

```diff
@@ -13,11 +13,15 @@ $check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -type "b
 
 $profiles = Get-AnsibleParam -obj $params -name "profiles" -type "list" -default @("Domain", "Private", "Public")
 $state = Get-AnsibleParam -obj $params -name "state" -type "str" -failifempty $true -validateset 'disabled','enabled'
+$inbound = Get-AnsibleParam -obj $params -name "inbound" -type "str" -default "Block"
```

Comment:
I would probably lean more towards the option names being `inbound_action` and `outbound_action`. Removes any ambiguity as to what the option is actually for. Also

* The values should be lowercase, i.e. `block` and `allow`
* You should add `-validateset 'allow', 'block` so Ansible validates the input for you.

It also seems like we should have `not_configured` to allow people wanting to use GPO for this object.